### PR TITLE
Added ObserveIT Workflow

### PR DESCRIPTION
**Pull request by LiorQM**
Hi, 

I've created a workflow for ObserveIT and I wanted to add it to [IBM-QRadar-Universal-Cloud-REST-API](https://github.com/IBM/IBM-QRadar-Universal-Cloud-REST-API).

Thanks, 
Lior

### CODE REVIEW

**Comment by sarbqdev** [comment]
@LiorQM @ChrisCollinsIBM Please wait with margin (@ChrisCollinsIBM please review the connector for now and give some feedback :) ), we are still waiting for the Vendor Approval to publish this.

Thanks !

**Comment by LiorQM** [comment]
We've got the vendor's approval to publish this project.
@ChrisCollinsIBM I'd appreciate if you can review this and give me some helpful feedback! 🙂

File: Community Developed/ObserveIT/ObserveIT-Workflow.xml [hunk 1]
```diff
@@ -0,0 +1,241 @@
+<Workflow name="Qmasters ObserveIT" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V1">
+    <Parameters>
+		<Parameter name="hostname" label="Host Name" description="IP for the instance." required="true" default="myobserveit.com" />
```
Comment (ChrisCollinsIBM) [inline Community Developed/ObserveIT/ObserveIT-Workflow.xml:3]: The field is called "hostname" but the description asks for an IP.

Depending on what CN and SAN identifiers are issued to the certificate on this endpoint, users may run into some hostname validation issues with SSL.

If the certificate is issued to the hostname it should be clear that it's the hostname you're looking for.

**Review by ChrisCollinsIBM — APPROVED** [review]
Looks great overall, added a few review items that would be nice to have for a future update.

**Pull request merged**